docs/source/interfaces/Muon_Analysis.rst

```diff
@@ -372,12 +387,27 @@ Examples/Use cases
    - It is, of course, possible to select several runs, groups, periods all at once and a simultaneous fit will be performed across all the selected datasets.
    - Example: MUSR{15189, 15190, 15191}, groups {*fwd*, *bwd*}, periods {1, 2}: 12 datasets in all.
 
-Options
-^^^^^^^
-The bottom of the tab contains selected fit options that can be adjusted, just as elsewhere in Mantid.
+Settings
```

Comment:
I think Options was less confusing, I was looking for the mentioned options in the `Settings` tab. Maybe a longer name, something like:
- Data Analysis Options
- Additional Options

would be more clear, because there is already a Settings tab